psydac/feec/tests/test_commuting_projections.py

```diff
@@ -221,6 +252,27 @@ def test_3d_commuting_pro_3(Nel, Nq, p, bc, m):
     error = abs((Dfun_proj.coeffs-Dfun_h.coeffs).toarray()).max()
     assert error < 1e-9
 
+    # TODO: test takes too long in 3D
```

Comment:
As above

Reply:
Fixed in commit [8c2c79a](https://github.com/pyccel/psydac/pull/347/commits/8c2c79a73267304b2fbff8447a63b674e8366c29)